conf/defaults.ini

```diff
@@ -1434,6 +1434,11 @@ enable_alpha = false
 app_tls_skip_verify_insecure = false
 # Enter a comma-separated list of plugin identifiers to identify plugins to load even if they are unsigned. Plugins with modified signatures are never loaded.
 allow_loading_unsigned_plugins =
+# Enter a comma-separated list of plugin identifiers to identify core data source plugins to be enabled explicitly. All are enabled if not specified or empty.
+# If not empty, those that are not given will be never loaded and not shown in plugin catalog. 
+core_datasource_plugins_enabled =
```

Comment:
Why do we need an option specifically for the core plugins? Could this not be a generic option that can disable any plugin?

I know you can uninstall an external plugin but I think even in that case it could be useful to disable / hide it without having to uninstall it, and should make for a cleaner solution that does not need to have special treatment for the core plugins? 

I don't know how technically feasible this is but should be straight forward, but could be mistaken.